### DISCUSSION TITLE
Get notifications for comment/reply of Discussions in slack (or equivalent)

### DISCUSSION THREAD

**Discussion by zoshigayan** [community/community#56821] [Discussions]
### Select Topic Area

Product Feedback

### Body

Currently, we can receive Slack notifications for Discussion only when an Discussion is created or resolved (a reply is marked as answer), but I would like to be notified of all updates such as comments/replies.

Thanks!

ref: https://github.com/orgs/community/discussions/2844

**Comment by stephaniegiang** [comment]
We don't currently have a way to subscribe to a comment/reply in Slack. However, you can receive notifications for all discussions and comments created in a repository by using the repository watch settings under custom notifications. And that will send notifications via the preference that is set in your GitHub settings (eg. email, web notifications). [Here is the link to the GitHub docs on how you can set custom notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository).

**Comment by HideakiSago** [comment]
Discussion allows for replies, so when sending notifications to Slack, threads are nested, and I suspect that this is the issue.

How about a notification to GitHub mobile app?
At the moment, only mentions are working, and no notifications were received for comments or replies without mentions.

**Comment by masutaka** [comment]
FYI:
I use GitHub Actions to notify Slack of comments on Discussions.
Usage: https://github.com/route06/actions/blob/v2.3.0/.github/workflows/gh_discussion_comment_to_slack.yml#L12-L25
